src/audio/Playback.ts

```diff
@@ -61,6 +61,7 @@ export class Playback extends EventEmitter implements IDestroyable {
     private waveformObservable = new SimpleObservable<number[]>();
     private readonly clock: PlaybackClock;
     private readonly fileSize: number;
+    private isInReply: boolean;
```

Comment:
This shouldn't be tracked off the playback instance - we should be ensuring we're not queueing the playback at all from elsewhere.

From memory, we track the voice message order by pushing on mount, so we should be able to use a component property.